### DISCUSSION TITLE
Changes only be applied in the editing mode, not on the actual page

### DISCUSSION THREAD

**Discussion by MarekZhang** [tinacms/tinacms#4079] [Help] [answered]
Hi team, 

I'm new to tina. After following the "Go to production" section in the TIna doc, I deployed the tina starter on my personal server. 
I noticed that I'm able to successfully commit changes to the connected Github repository, but these modifications are only visible within the live editing mode and do not appear on the actual page. It seems that in editing mode, data is fetched from the Github repository, whereas the actual page retrieves data from the local files stored on the sever. However, I could'nt find any instructions addressing this issue.

![image](https://github.com/tinacms/tinacms/assets/49737031/65311f02-874c-4b8d-9775-edd83cd052bd)
![image](https://github.com/tinacms/tinacms/assets/49737031/74181257-4ed5-4414-8b6d-ddb8b077c983)
![image](https://github.com/tinacms/tinacms/assets/49737031/09d3b323-fab5-405c-ade9-b93950cf9ed9)

I would appreciate any help you provided.

**Comment by jamespohalloran** [comment]
What your branch set to in your config.tsx? If its using an environment variable like: `NEXT_PUBLIC_TINA_BRANCH`, and you have that set in Vercel/Netlify, then all your branches will pull from the same branch, which may be not what you're expecting.

**Reply by MarekZhang** [reply]
Hi James,

Thanks for the reply. Actually I deployed it on an AWS EC2 instance without using Vercel/Netlify. The `NEXT_PUBLIC_TINA_BRANCH` is set to be `main`. Whenever I save changes in the editor, they were all commited to the Github repo.

**Reply by kldavis4** [reply, marked as answer]
If you changes are being committed to and you aren't appearing in the production site, then it sounds like your CI process is not deploying the updated site.

**Reply by MarekZhang** [reply]
Thanks Kelly, it turns out that I need to build the CI by myself to trigger the deployment ot EC2.
